DDSView/ddsview.cpp

```diff
@@ -385,7 +385,11 @@ HRESULT InitDevice( const TexMetadata& mdata )
     if( FAILED( hr ) )
         return hr;
 
-    hr = g_pd3dDevice->CreateRenderTargetView( pBackBuffer, nullptr, &g_pRenderTargetView );
+    D3D11_RENDER_TARGET_VIEW_DESC vd;
+    ZeroMemory(&vd, sizeof(vd));
```

Comment:
``ZeroMemory`` is very old-school. With VS 2013 or later, you can just use ``D3D11_RENDER_TARGET_VIEW_DESC vd = {};`` to get zero-filled memory.

Reply:
Apparently I never cleaned up this sample. How embarrassing...

Reply:
No problem! Yes, I actually followed style of previous statements. Actually I needed this sample working since I can use it as a reference. If I open the same textures I linked with Visual Studio (I tried 2013-2015-2017) I get similar weird results. I'm going open a bug report with Visual Studio (any version) I get similar weird results. I'm going open a bug report against VS 2017 for this.